src/containers/SortingPanel.container.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import SortingPanel from '../components/SortingPanel'
+
+const mapStateToProps = state => ({
+    state
+});
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
If you do not supply your own mapDispatchToProps function or object full of action creators, the default mapDispatchToProps implementation just injects dispatch into your component’s props.

So, you can just pass null as a second argument and dispatch function will appear as a wrapped component props

I recommend you to read mapDispatchToProps docs for better understanding:
https://github.com/reduxjs/react-redux/blob/master/docs/api.md#arguments